src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  box-sizing: border-box;
+
+  &__link {
+    text-decoration: none;
+    color: $nav-link-color;
+    font-size: 12px;
+    position: relative;
+    display: flex;
+    margin: auto;
+    align-items: center;
+    text-transform: uppercase;
+    height: 60px;
+
+    &:hover {
+      color: $nav-link-hover-color;
+    }
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+}
+
+.is-active {
+  color: $nav-link-hover-color;
```

Comment:
The variable `$nav-link-hover-color` is used here, but it is not defined within this file. Ensure that this variable is defined in a global SCSS file or imported correctly to avoid compilation errors.